libpdb/client_config.go

```diff
@@ -0,0 +1,36 @@
+package libpdb
+
+import (
+	"encoding/json"
+	"github.com/ryscheng/pdb/common"
+	"io/ioutil"
+  "time"
+)
+
+type ClientConfig struct {
+	*common.CommonConfig `json:"-"`
+
+  // How often should Writes be made to the server
+	WriteInterval  time.Duration
```

Comment:
I think the main point there was to differentiate the time interval that clients would use to send requests into the server, with the time interval when the shards would force write-flushes of the database. Agreed that it cna be moved around.